layouts/partials/persistence/persistence_table_row.html

```diff
@@ -0,0 +1,18 @@
+{{ $data := .Site.Data.persistence.coverage }}
+
+{{ range $data }}
+<tr>
+    <!-- Service -->
+    <td>{{ .full_name }}</td>
```

Comment:
would it be possible to add a link to the service? like https://localstack-docs-preview-pr-1259.surge.sh/user-guide/aws/acm/